src/commands/module/build.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { cwd } from "node:process";
+
+import { color } from "@oclif/color";
+import { Args } from "@oclif/core";
+
+import { BaseCommand, baseFlags } from "../../baseCommand.js";
+import { buildModules } from "../../lib/buildModules.js";
+import { commandObj } from "../../lib/commandObj.js";
+import { initReadonlyModuleConfig } from "../../lib/configs/project/module.js";
+import {
+  MARINE_BUILD_ARGS_FLAG,
+  MARINE_BUILD_ARGS_FLAG_NAME,
+  MODULE_CONFIG_FULL_FILE_NAME,
+} from "../../lib/const.js";
+import { initCli } from "../../lib/lifeCycle.js";
+import { initMarineCli } from "../../lib/marineCli.js";
+import { input } from "../../lib/prompt.js";
+
+const PATH = "PATH";
+
+export default class Build extends BaseCommand<typeof Build> {
```

Comment:
It seems the `--no-input` flag suppresses output 
```
$ ~/Downloads/ll/fluence/bin/run.js module build --no-input effector
<no output>
$ ~/Downloads/ll/fluence/bin/run.js module build  effector
    Finished release [optimized] target(s) in 0.05s
```
fcli build: @fluencelabs/cli/0.15.25-new-modules-fd8363e-6427-1.0 from CI

Reply:
yeah, seems this logic was there for a long time for all external binaries that CLI calls. Doesn't look very correct to me as well. Removed it